website/project/metadata/null-hypothesis-significance-testing.json

```diff
@@ -0,0 +1 @@
 religious group because we are not going to draw any conclusions based on these analyses.", "required": false}, {"help_text": "", "registration_response_key": "block5-9", "block_type": "long-text-input", "display_text": "", "example_text": "", "required": false}, {"help_text": "What will you do should your data violate assumptions, your model not converge, or some otheranalytic problem arises?", "registration_response_key": null, "block_type": "question-label", "display_text": "Assumption Violation/Model Non-Convergence", "example_text": "When the distribution of the number of volunteering hours (gv111re) is significantly non-normal according to the Kolmogorov-Smirnov test and/or (b) the linearity assumption is violated (i.e., the points are asymmetrically distributed around the diagonal line when plotting observed versus the predicted values), we will log-transform the variable.", "required": false}, {"help_text": "", "registration_response_key": "block5-11", "block_type": "long-text-input", "display_text": "", "example_text": "", "required": false}, {"help_text": "Provide a series of decisions or tests about evaluating the strength, reliability, or robustness of your finding. This may include within-study replication attempts, additional covariates, cross-validation, applying weights, selectively applying constraints in an SEM context (e.g., comparingmodel fit statistics), overfitting adjustment techniques used, or some other simulation/sampling/bootstrapping method.", "registration_response_key": null, "block_type": "question-label", "display_text": "Reliability and Robustness Testing", "example_text": "To assess the sensitivity of our results to our selection criterion for outliers, we will run an additional analysis without removing any outliers.", "required": false}, {"help_text": "", "registration_response_key": "block5-13", "block_type": "long-text-input", "display_text": "", "example_text": "", "required": false}, {"help_text": "If you plan to explore your dataset to look for unexpected differences or relationships, describe those tests here. If reported, add them to the final paper under a heading that clearly differentiates this exploratory part of your study from the confirmatory part.", "registration_response_key": null, "block_type": "question-label", "display_text": "Exploratory analysis", "example_text": "As an exploratory analysis, we will test the relationship between scores on the religiosity scale and prosociality after adjusting for a variety of social, educational, and cognitive covariates that are available in the dataset. We have no specific hypotheses about which covariates will attenuate the religiosity-prosociality relation most substantially, but we will use this exploratory analysis to generate hypotheses to test in other, independent datasets.", "required": false}, {"help_text": "", "registration_response_key": "block5-15", "block_type": "long-text-input", "display_text": "", "example_text": "", "required": false}]}
```

Comment:
This is impossible to review, can you use pprint or something to output it in a way that's legible?